sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -73,6 +74,7 @@ class Analyzer(
       ResolveGroupingAnalytics ::
       ResolvePivot ::
       ResolveUpCast ::
+      ResolveAggregateFunctions ::
```

Comment:
Yeah, you are right. Will do the change to exclude the cases that can be resolved by `ResolveAggregateFunctions`. Thanks!